dist/Devel-PPPort/t/ppphtest.t

```diff
@@ -762,7 +762,7 @@ my %p;
 my $fail = 0;
 for (@o) {
   my($name, $ver) = /^(\w+)\s*\.+\s*([\d._]+)$/ or $fail++;
-  exists $p{$name} and $fail++;
+  { exists $p{$name} and $fail++; }
```

Comment:
Why is this extra scope required?

Reply:
ditto

Reply:
this will be solved in the next D-PPP update